src/node_contextify.cc

```diff
@@ -387,11 +387,15 @@ class ContextifyContext {
     Local<Object> proxy_global = PersistentToLocal(isolate,
                                                    ctx->proxy_global_);
 
-    bool in_sandbox = sandbox->GetRealNamedProperty(property).IsEmpty();
+    bool in_sandbox = !(sandbox->GetRealNamedProperty(property).IsEmpty());
     bool in_proxy_global =
-        proxy_global->GetRealNamedProperty(property).IsEmpty();
-    if (!in_sandbox || !in_proxy_global) {
-      args.GetReturnValue().Set(None);
+      !(proxy_global->GetRealNamedProperty(property).IsEmpty());
+    if (in_sandbox || in_proxy_global) {
+      v8::PropertyAttribute attr = None;
+      if (in_sandbox) {
+        attr = sandbox->GetPropertyAttributes(property);
+      }
```

Comment:
Shouldn't we also add `if (in_proxy_global) { attr = proxy_global->GetPropertyAttributes(property); }` as the fallback?